tests/symbolicator/__init__.py

```diff
@@ -26,10 +26,20 @@ def strip_frame(frame):
     return frame
 
 
+NORMALIZED_REGISTERS = {"eflags": "efl"}
+
+
+def normalize_register(name):
+    return NORMALIZED_REGISTERS.get(name) or name
```

Comment:
fwiw in python this is written as `NORMALIZED_REGISTERS.get(name, name)` where the 2nd argument is the default value to return if the item is not present in the dict (and the default default item is `None`).